tests/Resources/test_info.py

```diff
@@ -1,4 +1,8 @@
+import os
 import unittest
+from pathlib import Path
+
+from flask import json
```

Comment:
we should probably just use the standard library json import here for consistency